plugins/utils/mini.nix

```diff
@@ -82,9 +90,13 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
   callSetup = false;
   hasSettings = false;
   extraConfig = cfg: {
```

Comment:
I think we need an assertion, to ensure the option is used correctly:

`assertion = cfg.mockDevIcons -> cfg.mocules ? icon`